patch away LC_NUMERIC setting; it results in failure while generating
pods with perl 5.38:

PERL5LIB=/usr/local/libdata/perl5/site_perl /usr/bin/perl -I../thirdparty/lib/perl5 -I../lib -mSmokeping -e 'Smokeping::main()' -- --makepod Smokeping::probes::AnotherCurl > Smokeping_probes_AnotherCurl.pod
Unknown locale category 4 at ../lib/Smokeping.pm line 40.
Segmentation fault (core dumped) 

Code shouldn't be needed on OpenBSD anyway, it's there to take care of
numeric locales with , instead of . (etc) and we don't do those anyway.

Index: lib/Smokeping.pm
--- lib/Smokeping.pm.orig
+++ lib/Smokeping.pm
@@ -37,36 +37,6 @@ setlogsock('unix')
 my $xssBadRx = qr/[<>%&'";]/;
 
 $ENV{'LC_NUMERIC'}='C';
-if (setlocale(LC_NUMERIC,"") ne "C") {
-    if ($ENV{'LC_ALL'} eq 'C') {
-        # This has got to be a bug in perl/mod_perl, apache or libc
-        die("Your internationalization implementation on your operating system is "
-          . "not responding to your setup of LC_ALL to \"C\" as LC_NUMERIC is "
-          . "coming up as \"" . setlocale(LC_NUMERIC, "") . "\" leaving "
-          . "smokeping unable to compare numbers...");
-    }
-    elsif ($ENV{'LC_ALL'} ne "") {
-        # This error is most likely setup related and easy to fix with proper 
-        # setup of the operating system or multilanguage locale setup.  Hint,
-        # setting LANG is better than setting LC_ALL...
-        die("Resetting LC_NUMERIC failed probably because your international "
-          . "setup of the LC_ALL to \"". $ENV{'LC_ALL'} . "\" is overriding "
-          . "LC_NUMERIC.  Setting LC_ALL is not compatible with smokeping...");
-    }
-    else {
-        # This is pretty nasty to figure out.  Seems there are still lots
-        # of bugs in LOCALE behavior and if you get this error, you are
-        # affected by it.  The worst is when "setlocale" is reading the
-        # environment variables of your webserver and not reading the PERL
-        # %ENV array like it should.
-        die("Something is wrong with the internationalization setup of your "
-          . "operating system, webserver, or the perl plugin to your webserver "
-          . "(like mod_perl) and smokeping can not compare numbers correctly.  "
-          . "On unix, check your /etc/locale.gen and run sudo locale-gen, set "
-          . "LC_NUMERIC in your perl plugin config or even your webserver "
-          . "startup script to potentially fix or work around the problem...");
-    }
-}
 
 
 use File::Basename;
